pytealutils/storage/blob/blob.py

```diff
@@ -0,0 +1,206 @@
+from pyteal import (
+    Extract,
+    GetByte,
+    SetByte,
+    Concat,
+    Substring,
+    Len,
+    Or,
+    Itob,
+    BytesZero,
+    Subroutine,
+    Int,
+    TealType,
+    ScratchVar,
+    For,
+    App,
+    Expr,
+    Seq,
+    Bytes,
+    If,
+)
+
+max_keys = 16
+page_size = 128 - 1  # need 1 byte for key
+max_bytes = max_keys * page_size
+max_bits = max_bytes * 8
+
+maxKeys = Int(max_keys)
+pageSize = Int(page_size)
+maxBytes = int(max_bytes)
+
+
+@Subroutine(TealType.bytes)
+def intkey(i: TealType.uint64) -> Expr:
+    return Extract(Itob(i), Int(7), Int(1))
+
+
+class Blob:
+    """
+    Blob is a class holding static methods to work with the local storage of an account as a binary large object
+
+    The `zero` method must be called on an account on opt in and the schema of the local storage should be 16 bytes
+    """
+
+    def __init__(self):
+        # Add Keyspace range?
+        # Allow global storage option
+        pass
```

Comment:
If end up not passing any arguments, better to just omit completely

Reply:
I omitted it initially but reconsidered and added comments here for things we _might_ want to do. 

What do you think of the above? If I set instance vars like keyspace it gets slightly more complicated to call since the methods are no longer static

Reply:
You might want that functionality at which point you can add the `__init__()`, but until you do I recommend removing. I like your approach of just putting basic functionality out there following the 80-20 rule.